src/main/java/net/syscon/elite/repository/impl/SentenceRepositoryImpl.java

```diff
@@ -28,7 +28,7 @@
 
         return jdbcTemplate.query(
                 sql,
-                createParams("bookingId", bookingId, "mostSerious", "Y", "chargeStatus", "A", "severityRanking", 999),
+                createParams("bookingId", bookingId, "mostSerious", "Y", "chargeStatus", "A", "severityRanking", "999"),
```

Comment:
turns out that severity ranking is stored in oracle as
```
	SEVERITY_RANKING VARCHAR2(12 char),
```
so needs to be sent in as a string instead (okay in hsqldb, but oracle moans)